model/admin/settings.go

```diff
@@ -0,0 +1,30 @@
+package admin
+
+type AdminSettings struct {
+	Id                   string       `bson:"_id" json:"id"`
+	Banner               string       `bson:"banner" json:"banner"`
+	TaskDispatchDisabled bool         `bson:"task_dispatch_disabled" json:"task_dispatch_disabled"`
+	RunnerFlags          *RunnerFlags `bson:"runner_flags" json:"runner_flags"`
+}
+
+type RunnerFlags struct {
```

Comment:
Can this be "serviceflags" and then have TaskDispatchDisabled could easily live here?